src/components/Pagination/Pagination.scss

```diff
@@ -1,5 +1,7 @@
 $prefixClass: rc-pagination;
 
+$font-size : rem(12px);
```

Comment:
@Farhan-emgage I don't think this is necessary. See line 17 comment.

Reply:
See comment on Accordion.scss. Font size prop should be set once via a single class. Looks like application to `.rc-pagination` should be all that's needed.

Reply:
@Farhan-emgage TODO: Need to resolve during meeting.